reginald/parser_utils.py

```diff
@@ -0,0 +1,226 @@
+import argparse
+import pathlib
+
+from reginald.models.models import MODELS
+from reginald.utils import get_env_var
+
+
+class Parser(argparse.ArgumentParser):
+    def __init__(self, create_index_only: bool = False, *args, **kwargs):
+        """
+        Parser for command line arguments for Reginald.
+
+        Note that the default values for the arguments are set to a lambda function
+        to obtain the relevant environment variables (if they exist). They are
+        lambda functions so that the environment variables are only obtained if
+        the argument is actually used.
+
+        See `get_args` for parsing the arguments and obtaining any of the
+        environment variables if using default values.
+
+        Parameters
+        ----------
+        create_index_only : bool, optional
+            Whether or not to only include ones related to data index creation,
+            by default False
```

Comment:
```suggestion
            Whether or not to only include arguments related to data index creation,
```

Reply:
I'm a bit confused by this suggestion - would it mean repeating the "Whether or not to..." line?

Reply:
I'm going to stop using vs code to do suggestions, I think it should say "Whether or not to only include arguments related to..."